.travis.yml

```diff
@@ -98,6 +98,7 @@ jobs:
         - travis_wait make test-cmd-app
         - travis_wait make test-cmd-push
         - travis_wait make test-cmd-project
+        - travis_wait make test-cmd-devfile-push
```

Comment:
+1,
FYI - Travis job runs test on 3.11 cluster

Reply:
Update the [name:](https://github.com/openshift/odo/pull/2681/files#diff-354f30a63fb0907d4ad57269548329e3L90) field accordingly. For example 
```
name: "watch, storage, app, project, push and devfile push command integration tests"
```

Reply:
Good catch, updated!